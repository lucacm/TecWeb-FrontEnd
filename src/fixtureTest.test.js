import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react'
import { shallow ,mount } from 'enzyme';
import toJson from "enzyme-to-json"
import ReactTestUtils from 'react-dom/test-utils';
import Fixtures from './screens/fixtures';
import App from './App'


afterEach(cleanup)


it('Should filter the main screen and display with only Chelsea games', function () {
   const app = mount(<Fixtures/>)

   expect(app.find('input').simulate('change', {target: {value: "chelsea"}})).toMatchSnapshot()
  });
  