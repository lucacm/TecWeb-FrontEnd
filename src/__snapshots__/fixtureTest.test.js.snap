// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render the main screen with only Chelsea games 1`] = `ReactWrapper {}`;

exports[`Should filter the main screen and display games won by away team 1`] = `ReactWrapper {}`;

exports[`Should filter the main screen and display games won by home team 1`] = `ReactWrapper {}`;

exports[`Should filter the main screen and display tied games 1`] = `ReactWrapper {}`;

exports[`Should filter the main screen and display with only Chelsea games 1`] = `ReactWrapper {}`;
